### PR TITLE
Add terrain-profile to oskari-server-extras

### PR DESCRIPTION
**Pull request by jampukka**
Goal is to make
```
<groupId>fi.nls.paikkatietoikkuna</groupId>
<artifactId>service-terrain-profile</artifactId>
<version>1.21.0-SNAPSHOT</version>
```
available as a dependency for other applications (besides Paikkatietoikkuna) by publishing it under Oskari.org repository as
```
<groupId>fi.nls.oskari.extras</groupId>
<artifactId>service-terrain-profile</artifactId>
<version>1.0</version>
```

### CODE REVIEW

**Comment by ZakarFin** [comment]
Trying to compile with Maven it says:
```
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /C:/Omat/oskari/oskari-server-extras/service-terrain-profile/src/main/java/fi/nls/oskari/terrainprofile/dem/ScaledGrayscaleValueExtractor.java:[44,97] incompatible types: short[] cannot be converted to float[]
[ERROR] /C:/Omat/oskari/oskari-server-extras/service-terrain-profile/src/main/java/fi/nls/oskari/terrainprofile/dem/FloatAsIsValueExtractor.java:[30,77] incompatible types: bad return type in lambda expression
    void cannot be converted to float[]
[INFO] 2 errors
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------

File: service-terrain-profile/pom.xml [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+    <groupId>fi.nls.oskari.extras</groupId>
+    <artifactId>service-terrain-profile</artifactId>
+    <version>1.0</version>
+    <packaging>jar</packaging>
+    <name>Terrain Profile</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <jdk.version>1.8</jdk.version>
+        <oskari.version>[2.4.0,)</oskari.version>
```
Comment (ZakarFin) [inline service-terrain-profile/pom.xml:15]: Lets' bump up the required version:
```suggestion
        <oskari.version>[2.6.0,)</oskari.version>
```

**Comment by ZakarFin** [comment]
Just for digging up history the original version is from here: https://github.com/nls-oskari/kartta.paikkatietoikkuna.fi/tree/0e55d5123f5c518c4d9066886d5336337aacb4d4/service-terrain-profile

**Comment by jampukka** [comment]
Just remembered that this actually requires https://github.com/oskariorg/oskari-server/pull/813 which is not part of even 2.6.0 but upcoming 2.7.0

**Comment by ZakarFin** [comment]
Awesome, thanks! I'll merge this now that 2.7.0 Oskari-server has been released.

**Pull request merged**